### PR TITLE
Updated Theme Link and added favicon

### PR DESCRIPTION
**Pull request by bmorgan296**
Updated the link to theme-dev.byu.edu instead of theme.byu.edu and asked for any suggestions for that site. Also include a note about the favicon designed by Aleni. Is it possible to get the icon file posted in GitHub instead of a png?

# Summary of Changes

**Fixes Issue #165**

*What did you change? If this is a bug fix, how did you fix it?*
Updated the readme to include the link to theme-dev and the favicon.
**If this fixes styling, please include before and after screenshots!**

# Browser Testing

**I have tested these changes in:**

*Add an x in all the boxes that apply. Please mark desktop and mobile
browsers separately.*

## Desktop Browsers

- [ ] Google Chrome
- [ ] Mozilla Firefox
- [ ] Apple Safari
- [ ] Microsoft Edge
- [ ] Microsoft Internet Explorer 11
- [ ] Other (please specify)

## Mobile Browsers

- [ ] Any browser on iOS
- [ ] Chrome for Android
- [ ] Firefox Mobile for Android
- [ ] Other (please specify)

**We support the last two versions of Chrome, Firefox, Safari, and Edge,
plus Internet Explorer 11.**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,5 +40,8 @@ Our goal is to implement flexible, well-structured web components according to t
 
 In addition, every component must be responsive, providing a first-class experience to phone and tablet users.
 
+## Favicon
+All BYU Websites are encouraged to use the official favicon. This is likely to be located on brand.byu.edu in the future, but it is available now in github: [https://github.com/byuweb/byu-assets/blob/master/favicon.png](https://github.com/byuweb/byu-assets/blob/master/favicon.png).
+
```
Comment (kebowdog) [inline README.md]: I'd like to change this so the .ico file is hosted, not the png. Pointing people to a png won't really help them get the correct favicon. I've asked @katrialesser for push access to the byu-assets repo so I can upload the correct favicon.

Comment (kebowdog) [inline README.md]: Also, can you add the favicon to the docs folder so the components demo site has it as well?

Comment (ThatJoeMoore) [inline README.md]: @kebowdog Why do you want the .ico?  All modern browsers support .png favicons, and .ico kind of really sucks.

Comment (ThatJoeMoore) [inline README.md]: https://en.wikipedia.org/wiki/Favicon#File_format_support

Comment (kebowdog) [inline README.md]: This is an example of me not doing my homework. I happily withdraw my request for changes.

**Pull request merged**